modules/ROOT/pages/workspaces-and-repositories.adoc

```diff
@@ -53,6 +53,14 @@ The exported archive can then be shared with other studios of the same version o
 
 To import a BOS archive:
 
+[WARNING]
+====
+
+Starting with version 2021.2, Java versions lower than Java 11 are no longer supported. 
+As a result, some of the dependendies in the projects imported from previous Bonita Studio versions may not be compatible with Java 11 (for example, an old dependency may contain some packages that have been moved into the JDK in Java 11, which could lead to issues with modules as the same package might be loaded twice).
```

Comment:
```suggestion
As a result, some of the dependencies in the projects imported from previous Bonita Studio versions may not be compatible with Java 11 (for example, an old dependency may contain some packages that have been moved into the JDK in Java 11, which could lead to issues with modules as the same package might be loaded twice). +
This typically leads to the following error: `The package XXX is accessible from more than one module: <unnamed>, YYY`
```

Reply:
Good catch!
And good additional line.
Thanks